lib/webpacker/dev_server.rb

```diff
@@ -21,11 +21,11 @@ def hot_module_replacing?
   end
 
   def host
-    fetch(:host)
+    ENV["WEBPACK_DEV_SERVER_LISTEN_HOST"] || fetch(:host)
```

Comment:
These should all match their method names, e.g. `ENV["WEBPACK_DEV_SERVER_HOST"]`